assembl/models/attachment.py

```diff
@@ -167,6 +175,10 @@ def __init__(self, *args, **kwargs):
 
     # Should we defer this?
     data = Column(LargeBinary, nullable=False)
+    # Note: use SQLA 1.1 for a better Enum behaviour
+    av_checked = Column(Enum(*AntiVirusStatus.__members__.keys(),
+                             name="anti_virus_status"),
+                        server_default='unchecked')
```

Comment:
instead of `server_default='unchecked'`, you could probably use something like `server_default=AntiVirusStatus.unchecked` for safety